node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+
+fs.writeFile('random.txt', `${Math.random().toString()} \n`, 'utf8', err => {
```

Comment:
You don't need to manually call `.toString()` if something is passed to a template literal, it will automatically convert it.